### PR TITLE
Verification of HP ALM 

### PR DESCRIPTION
**Pull request by lmontopo**
Add field scopes and missing fields.

### CODE REVIEW

File: sdetools/modules/sync_hp_alm/hp_alm_metadata.py [hunk 1]
```diff
@@ -25,69 +25,61 @@ def supports_references(self):
     def __init__(self, phases):
         super(HpAlmMetadata, self).__init__(phases)
 
-        # update server field
-        self.alm_server.default = ''
-
```
Comment (lmontopo) [inline sdetools/modules/sync_hp_alm/hp_alm_metadata.py]: any idea what we should put here @simon? or just leave it empty? or with the 'bugs.example.com'?

File: sdetools/modules/sync_hp_alm/hp_alm_metadata.py [hunk 2]
```diff
@@ -25,69 +25,61 @@ def supports_references(self):
...
             field_type=FieldType.STRING,
-            default='Functional',
-            placeholder='Functional'
-        )
-        self.hp_alm_done_statuses = IntegrationField(
-            name='hp_alm_done_statuses',
-            label=_('States which map to DONE in SD Elements:'),
-            field_type=FieldType.STRING,
-            placeholder='Passed',
-            default='Passed'
```
Comment (lmontopo) [inline sdetools/modules/sync_hp_alm/hp_alm_metadata.py]: i just moved this field down below, FYI.

File: sdetools/modules/sync_hp_alm/hp_alm_metadata.py [hunk 3]
```diff
@@ -25,69 +25,61 @@ def supports_references(self):
...
-            field_type=FieldType.STRING,
-            placeholder='SD Elements',
-            default='SD Elements',
-        )
-        self.hp_alm_test_type = IntegrationField(
-            name='hp_alm_test_type',
-            label=_('Default type for test plans'),
-            field_type=FieldType.STRING,
-            placeholder='MANUAL',
-            default='MANUAL',
```
Comment (lmontopo) [inline sdetools/modules/sync_hp_alm/hp_alm_metadata.py]: i didn't see `hp_alm_test_plan_folder` or `hp_alm_test_type` in our app curretly.

File: sdetools/modules/sync_hp_alm/hp_alm_metadata.py [hunk 4]
```diff
@@ -25,69 +25,61 @@ def supports_references(self):
...
+            scope=['system']
         )
         self.hp_alm_domain = IntegrationField(
             name='hp_alm_domain',
             label=_('Domain'),
             field_type=FieldType.STRING,
-            placeholder='',
-            default='',
-        )
-        self.hp_alm_test_plan_folder = IntegrationField(
```
Comment (sibartlett) [inline sdetools/modules/sync_hp_alm/hp_alm_metadata.py]: Are these fields not in the old UI?

Comment (lmontopo) [inline sdetools/modules/sync_hp_alm/hp_alm_metadata.py]: correct, didn't see em at the system or project level.

File: sdetools/modules/sync_hp_alm/hp_alm_metadata.py (hunk 1 above)
Comment (sibartlett) [inline sdetools/modules/sync_hp_alm/hp_alm_metadata.py]: No need to override it the default for HP ALM

**Pull request merged**